### DISCUSSION TITLE
npm package 0.18.0 have invalid anchor binary

### DISCUSSION THREAD

**Discussion by Wuffle1125** [community/community#20851] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello @everyone.
I have this error

$ npm ls @project-serum/anchor-cli
solana-protocol@0.2.0 /home/kirill/projects/x-project
└── @project-serum/anchor-cli@0.18.0

$ npx anchor --version
Package binary version is not correct. Expected "anchor-cli 0.18.0", found "anchor-cli 0.17.0".
Trying globally installed anchor.
Failed to get version of global binary: Error: spawnSync undefined/anchor ENOENT

Please help me.

**Comment by devlancer412** [comment, marked as answer]
C:\WINDOWS\system32>npx anchor --version
Only x86_64 / Linux distributed in NPM package right now.
Trying globally installed anchor.
Failed to get version of global binary: Error: spawnSync \Users\draco\AppData\Roaming\npm/anchor ENOENT

hey help me out, i new to solana , I am getting above error ever installing cli globally by using cmd
npm i -g @project-serum/anchor-cli
still getting an error.
